README.md

```diff
@@ -45,3 +66,32 @@ Open the game in a new tab, then run `2048.py -b firefox` and watch the game! Th
 Enable Chrome remote debugging by quitting it and then restarting it with the `remote-debugging-port` command-line switch (e.g. `google-chrome --remote-debugging-port=9222`).
 
 Open the game in a new tab, then run `2048.py -b chrome` and watch the game! The `-p` option can be used to set the port to connect to.
+
+## Contributing
+
+We welcome contributions! If you have found a bug or want to suggest a new feature, please raise an issue.
+
+### Guidelines for contributing
+
+1. Fork the repository
+
+2. Clone your fork locally:
+
+	git clone https://github.com/your-username/2048-ai.git
+
+3. Create a new branch for new feature or big fix:
+
+	git checkout -b feature-branch
+
+4. Make your changes, commit, and push:
+
+	git add .
+	git commit -m "Your detailed commit message"
+	git push origin feature-branch
+
+5. Submit the pull request and we will review it!
+
+Remember to test your changes thoroughly to ensure they work as expected. If applicable, include unit tests to verify the functionality of your contribution. Make sure existing tests pass and your changes don’t introduce regressions. Your help is greatly appreciated!
```

Comment:
We aren't really doing tests in this repository, although a PR adding a test framework would be rather welcome. Maybe we can do something like StockTest does for Stockfish, where we only accept commits that improve the AI's performance.